deployment/v3/external/postgres/init_db.sh

```diff
@@ -8,14 +8,14 @@ fi
 
 function initialize_db() {
   NS=postgres
-  CHART_VERSION=12.0.1-B3
+  CHART_VERSION=12.0.2
   helm repo update
   while true; do
       read -p "CAUTION: all existing data will be lost. Are you sure?(Y/n)" yn
       if [ $yn = "Y" ]
         then
-          echo Removing any existing installation
-          helm -n $NS delete postgres-init
+          #echo Removing any existing installation
+          #helm -n $NS delete postgres-init
```

Comment:
Please uncomment this as we have already informed that data will be lost if you run the script

Reply:
changed as per requirment